NitroxClient/GameLogic/Spawning/DefaultEntitySpawner.cs

```diff
@@ -53,11 +53,15 @@ public Optional<GameObject> Spawn(Entity entity, Optional<GameObject> parent, En
 
         private GameObject CreateGameObject(TechType techType, string classId)
         {
-            IPrefabRequest prefabRequest = PrefabDatabase.GetPrefabAsync(classId);
-            if (!prefabRequest.TryGetPrefab(out GameObject prefab))
+            GameObject prefab = null;
+            if (PrefabDatabase.TryGetPrefabFilename(classId, out string filename))
             {
-                prefab = CraftData.GetPrefabForTechType(techType, false);
+                prefab = Resources.Load<GameObject>(filename);
+            }
```

Comment:
Maybe we should implement both CreateGameObject and CreateGameObjectAsync, so we can switch easily between them

Reply:
This PRis just to fix the sync method. 

I will be making a full overhaul PR for the spawning system separately.